.github/workflows/build_and_test.yml

```diff
@@ -242,6 +256,14 @@ jobs:
           source ${{ env.BUILD_ROOT }}/kong-dev-venv.sh
           .ci/run_tests.sh
 
+    - name: Upload test rerun information
```

Comment:
What happens when the first CI run yields some failures, but the second CI run succeeds? Is there some mechanism that "resets" the failed test list?

Reply:
The failed tests file is now unconditionally created, so when the failing test(s) pass(es), it will be empty.